package.json

```diff
@@ -30,7 +30,7 @@
     "is-git-url": "^1.0.0",
     "js-yaml": "^3.13.1",
     "lodash.pick": "^4.4.0",
-    "mesg-js": "^4.1.0",
+    "mesg-js": "file:../mesg-js",
```

Comment:
update to use a beta version on npm

Reply:
I spend 15min to properly install mesg-js.... can you push on npm with beta tag?